jooby/src/main/java/io/jooby/DefaultContext.java

```diff
@@ -228,10 +228,10 @@
   }
 
   @Override default @Nonnull String getRequestURL() {
-    return getRequestURL("");
+    return getRequestURL(null);
   }
 
-  @Override default @Nonnull String getRequestURL(@Nonnull String path) {
```

Comment:
let's keep the `@Nonnull` annotation. This always append the provided path (never the `getRequestPath` and never append the queryString

Reply:
The annotation is still in the interface (for external consumers), but locally, there was a null check already, so why not pass a null as "undefined" value? Empty string can actually have a meaning of empty path